scalability_and_performance/sno-du-connected.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: ASSEMBLY
+[id="sno-du-deploying-distributed-units-manually-on-single-node-openshift-connected"]
+= Deploying distributed units manually on single node OpenShift in a connected environment
+include::_attributes/common-attributes.adoc[]
+:context: sno-du-deploying-distributed-units-manually-on-single-node-openshift-connected
+
+toc::[]
+
+
+The procedures in this topic tell you how to manually deploy clusters on a small number of single nodes as a distributed unit (DU) during installation in a connected environment.
+
+The procedures do not describe how to install single node OpenShift (SNO). This can be accomplished through many mechanisms. Rather, they are intended to capture the elements that should be configured as part of the installation process:
+
+* Networking is needed to enable connectivity to the SNO DU when the installation is complete.
```

Comment:
@imiller0 @yliu127 Yang - please provide the "initial setup" information you were talking about. We need a better intro to the "connected environment" - why they want to do this, any deviations to installation/configuration, etc.